.github/workflows/go.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.22.3
+        go-version: 1.22.4
```

Comment:
test/localenv/README.md change the go installation here.

Reply:
I think already changed go version in test/localenv/README?